src/components/clients/ClientDetails.js

```diff
@@ -3,16 +3,37 @@ import React from 'react';
 class ClientDetails extends React.Component {
   render() {
     const client = this.props.client;
+    if (!client) {
+      return null;
+    }
```

Comment:
Yes, we don't render anything :)

This is just the box that appears when a new client is added. It should only be rendered in this case.